lib/linear_gauge/linear_gauge_painter.dart

```diff
@@ -467,9 +486,21 @@ class RenderLinearGauge extends RenderBox {
   void _drawPrimaryRulers(Canvas canvas) {
     _setPrimaryRulersPaint();
     _linearGaugeLabel.getPrimaryRulersOffset.forEach((key, value) {
-      double y = value[1].dy + getLinearGaugeBoxDecoration.height;
-      double x = value[1].dx;
+      // double y = value[1].dy + getLinearGaugeBoxDecoration.height;
```

Comment:
Need to understand what exactly this is doing ?

Reply:
// in case of invert labels
// y co-ordinate will be simply inverted on negative side by adding -ve sign
// no need to adjust y co-ordinate by adding the height of gauge container

// in case of non-invert labels
// there is need to adjust y co-ordinate by adding the height of gauge container
// bcz line will start drawing from behind of gauge container